core/src/main/scala/scala/pickling/pickler/Date.scala

```diff
@@ -15,7 +15,7 @@ trait DatePicklers extends PrimitivePicklers {
     }
     private def dateFormat = dateFormatTemplate.clone.asInstanceOf[SimpleDateFormat]
 
-    def tag = FastTypeTag[Date]
+    def tag = FastTypeTag[Date]("java.util.Date")
```

Comment:
OK, great. Makes sense now. I agree, an artifact for FastTypeTag would be good at some point.